extensions/2.0/Vendor/EXT_feature_metadata/README.md

```diff
@@ -354,20 +328,15 @@ _Class and feature texture_
           }
         }
       },
-      "featureTextures": {
-        "heatLossTexture": {
-          "class": "heatSample",
-          "properties": {
-            "heatLoss": {
-              "texture": {
-                "index": 0,
-                "texCoord": 0
-              },
-              "channels": "r"
-            }
-          }
+      "featureTextures": [{
+        "class": "heatSample",
+        "index": 0,
+        "texCoord": 0,
+        "properties": {
+          "heatLoss": [0],
+          "insulation": [1]
```

Comment:
Update the class definition so that it has `heatLoss` and `insulation` properties. Looks like we had a typo before too. The `heatSample` property should have been called `heatLoss`.